### DISCUSSION TITLE
Classify top developers of a organization

### DISCUSSION THREAD

**Discussion by Joelciomatias** [community/community#81934] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I want list the activities of the developers of a organization (i am a member of this organization).
First a need to list the members, after that i want the activities, number of commits, PR opened, releases, ect, and create a table of these data.
Which auth type i need? and which endpoints will useful for this?

**Comment by geeky01adarsh** [comment]
If the case is for repository (which is not your case, but I am sure it may help)
Try using the pulse, provided in the insights. Refer to this [Github Documentation](https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/using-pulse-to-view-a-summary-of-repository-activity) for more information.
